forum/qa-content/qa-page.js

```diff
@@ -582,192 +582,100 @@ function qa_ajax_error()
      * Feature: Post content preview as Modal
      * Date: 07.07.2016r.
      */
-    function postPreview(ckeCurrentInstance, placeForBtn)
+
+    function destroyModal()
     {
-        var modalParent = document.querySelector('.qa-main-wrapper');
+        const modal = document.querySelector('.post-preview-parent');
+        modal.parentNode.removeChild(modal);
 
-        var modalBackground = document.createElement('div');
-        modalBackground.classList.add('modal-background');
+        const modalBackground = document.querySelector('.modal-background');
+        modalBackground.parentNode.removeChild(modalBackground);
+    }
 
+    function createPostPreviewModal(ckeInstanceName)
+    {
+        const modal = document.createElement('div');
+        modal.classList.add('post-preview-parent');
 
-        var showModalBtn = document.createElement('input');
-        showModalBtn.type = 'button';
-        showModalBtn.id = 'get-content-preview';
-        showModalBtn.value = 'Podgląd posta';
-        showModalBtn.classList.add('qa-form-tall-button', 'get-content-preview');
+        const closeModalButton = document.createElement('button');
+        closeModalButton.innerHTML = 'X';

```

Comment:
Jeśli zawartością elementu jest tylko tekst, to bardziej nada się `closeModalButton.textContent`

Reply:
Zasugerowałem się Twoim zapisem z dotychczasowej wersji pluginu, przez to użyłem takiego zapisu. Ale dziękuję, oczywiście zamienię.